src/App/index.jsx

```diff
@@ -1,12 +1,19 @@
 import { FluentProvider, webLightTheme } from '@fluentui/react-components';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import AccountsList from '../AccountsList';
+import Account from '../Account';
 
 function App() {
   return (
     <FluentProvider theme={webLightTheme}>
-      <AccountsList />
+      <Router>
+        <Routes>
+          <Route path="/" element={<AccountsList />} />
+          <Route path="/accounts/:id" element={<Account />} />
+        </Routes>
+      </Router>
     </FluentProvider>
-  )
+  );
 }
 
-export default App;
+export default App;
```

Comment:
Put the export on the same line as the declaration on line six like this:

`export default function App`